modules/logging-release-notes-5-7-12.adoc

```diff
@@ -0,0 +1,53 @@
+// module included in logging/logging-5-7-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-7-12_{context}"]
+= Logging 5.7.12
+This release includes link:https://access.redhat.com/errata/RHSA-2024:1508[Logging Bug Fix 5.7.12].
+
+[id="logging-release-notes-5-7-12-bug-fixes"]
+== Bug fixes
+
+* Before this update, the Loki Operator checked whether the pods were running to decide if the `LokiStack` was ready. With this update, it also checks if the pods are ready, so the readiness of the `LokiStack` reflects the state of its components. (link:https://issues.redhat.com/browse/LOG-5172[LOG-5172])
+
+
+* Before this update, the Red Hat build pipeline didn't use the existing build details in Loki builds, leaving out information like revision, branch, and version. With this update, the Red Hat build pipeline now adds these inputs to the Loki builds, fixing the issue. (link:https://issues.redhat.com/browse/LOG-5202[LOG-5202])
```

Comment:
```suggestion
* Before this update, the Red Hat build pipeline didn't use the existing build details in Loki builds, leaving out information like revision, branch, and version. With this update, the Red Hat build pipeline now adds these details to the Loki builds, fixing the issue. (link:https://issues.redhat.com/browse/LOG-5202[LOG-5202])
```

Reply:
Not sure if "inputs" is the same as the "existing details" mentioned earlier in the previous sentence, so not sure if they are interchangeable?